vignettes/network-analysis.Rmd

```diff
@@ -0,0 +1,126 @@
+---
+title: "Introduction to Organizational Network Analysis with wpa in R"
+author: "Julia Kasper"
+date: "2021-08-18"
+categories: ["R"]
+tags: ["network analysis"]
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(collapse = TRUE)
+```
+
+This article provides a beginner’s introduction to Organizational Network Analysis (ONA), its basic concepts, and how to run them with the functions available from the [**wpa** R library](https://microsoft.github.io/wpa/).
+
+## Introduction
+
+**Organizational Network Analysis (ONA)**, in the context of Workplace Analytics, is a family of methods to understand patterns of collaboration by examining the strength, frequency and nature of interactions between people. Analyzing and visualizing network connectivity in your organization can help you shape business strategy that unlocks success for individual and team productivity, innovation, employee engagement and organizational change, thereby helping your business become more resilient and effective.
+
+Most valuable information in organizations is not codified. As employees collaborate, information flows from one person to the other or they generate knowledge together. These connections deliver value when information is exchanged and visualizing these relationships can provide us with a holistic view of how information flows through an organization. ONA can help us reveal central network collaborators, critical connections, and potential barriers to information flow.
+
+### Basic Building Blocks
+
+We will start with the basic building blocks of networks. Networks consist of two primary aspects, a multitude of separate entities ("nodes") and the connections between them ("edges"). Nodes and edges are the foundation for understanding how information in your organization is flowing, can flow, and should flow. Nodes in ONA visualizations typically represent individual employees or groups of employees (e.g. `Organization`, `Region` etc.), serving as important hubs for the exchange of ideas and information. In addition to these nodes, you have connections (so-called "edges") between them. These edges indicate there is some relationship between the nodes, meaning these individuals or groups have a connection to each other when they collaborate. In the context of Workplace Analytics, we identify these connections between nodes in the network based on collaboration activities (emails, meetings, calls, instant messages). In a network analysis, we also need to understand the strength of the tie. For this calculation we are using time spent together in collaboration. In addition, we have to consider the **direction** in which collaboration is operated. For meetings there is no direction, but for emails, chats and calls we have a sense of direction which we can look at.
```

Comment:
@juliajuju93 The line on `is flowing, can flow, and should flow` sounds slightly excessive on what you can do from a graph. What a graph really tells you is how information _is flowing_, but the other two depends on factors that lie entirely outside the graph. Perhaps can substantiate this to say something alone the lines of, `nodes and edges provide a foundation on how information is flowing, and also strategize on how they can flow and should flow` ?